### PR TITLE
feat(kafka-node.js): support kafka-node

### PR DESCRIPTION
**Pull request by ranrib**

### CODE REVIEW

File: src/events/kafka-node.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        } catch (err) {
+            tracer.addException(err);
+        }
+
+        const responsePromise = new Promise((resolve) => {
+            patchedCallback = (err, data) => {
+                let callbackResult;
+                try {
+                    if (callback && typeof callback === 'function') {
+                        callbackResult = callback(err, data);
```
Comment (galbash) [inline src/events/kafka-node.js]: better call finalize event before callback (for time measurement)

Comment (galbash) [reply src/events/kafka-node.js]: maybe just call the callback in a `finally` clause

**Review by galbash — CHANGES_REQUESTED** [review]
few small comments

File: src/events/kafka-node.js (hunk 1 above)
Comment (ranrib) [reply src/events/kafka-node.js]: 👍

File: src/events/kafka-node.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                    );
+                } catch (callbackErr) {
+                    tracer.addException(callbackErr);
+                } finally {
+                    if (callback && typeof callback === 'function') {
+                        callbackResult = callback(err, data);
+                    }
+                }
+                resolve();
+                return callbackResult;
```
Comment (galbash) [inline src/events/kafka-node.js:94]: you can move the `resolve()` to the finally and then you won't need `callbackResult` at all

**Review by galbash — APPROVED** [review]

**Pull request merged**